docs/guides/admin/context-links.md

```diff
@@ -0,0 +1,139 @@
+# Context Links
+
+This feature was introduced as part of [#2439](https://github.com/google/timesketch/issues/2439) and allows linking of specific event attributes and values to external lookup services for an easy analyst workflow.
+
+## Demo
+
+![TS_ContextLinks_RedirectWarn](https://user-images.githubusercontent.com/99879757/205897124-00154a89-e58a-4cf0-a89e-ba3b4f3052d8.gif)
+
+## How-To configure
+
+The context link feature is configured in a yaml file that needs to be added to the [timesketch.conf](https://github.com/google/timesketch/blob/master/data/timesketch.conf#L325) file. Add the path to the yaml config file in the `CONTEXT_LINKS_CONFIG_PATH` variable.
+
+Per default the `CONTEXT_LINKS_CONFIG_PATH` entry in `timesketch.conf` points to the [context_links.yaml](https://github.com/google/timesketch/blob/master/data/context_links.yaml) file in the `data` folder. It contains a documentation of available fields and a commented example entry for setting up a hash-lookup with [virustotal](https://www.virustotal.com/).
+
+**Important: You can add a context link only for services that allow a lookup via URL GET parameter.**
+
+### Configuration fields
+
+Each context link consists of the following fields:
+
+```
+context_link_name:
+
+  short_name:       Type: str | The name for the context link.
+                    Will be displayed in the context link submenu.
+
+  match_fields:     Type: list[str] | List of field keys where
+                    this context link should be available. Will
+                    be checked as case insensitive!
+
+  validation_regex: Type: str | OPTIONAL
+                    A regex pattern that needs to be
+                    matched by the field value to to make the
+                    context link available. This can be used to
+                    validate the format of a value (e.g. a hash).
+
+  context_link:     Type: str | The link that will be opened in a
+                    new tab when the context link is clicked.
+                    IMPORTANT: Add the placeholder "<ATTR_VALUE>"
+                    where the attribute value should be inserted
+                    into the link.
+
+  redirect_warning: [TRUE]: If the context link is clicked it will
+                            open a pop-up dialog first that asks the
+                            user if they would like to proceed to
+                            the linked page. (Recommended for
+                            external pages.)
+                    [FALSE]: The linked page will be opened without
+                             any pop-up. (Recommended for internal
+                             pages.)
+```
+
+### Test configuration
+
+If we want to test our configuration we can use the
+`tsctl validate-context-links-conf` > [here](https://github.com/google/timesketch/blob/master/docs/guides/admin/admin-cli.md#context-links).
```

Comment:
can you iinstead link to it works for timesketch.org?

Reply:
Good point. I added a link to timesketch.org and hope it works like this when merged :)